INSTALL_WINDOWS.md

```diff
@@ -0,0 +1,17 @@
+# Prerequisites on Windows
+
+1. Install [Visual C++ Build Tools](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=BuildTools&rel=16): select MSVC Build Tools + Windows 10 SDK
+2. Install [WIX](https://github.com/wixtoolset/wix3/releases/download/wix3112rtm/wix311.exe)
+3. Copy [wixgen.jar](https://github.com/akashche/wixgen/releases/download/1.7/wixgen.jar) to `C:\msi-deps`
+4. Copy all 65 files from `C:\Program Files (x86)\WiX Toolset v3.11\bin` to `C:\msi-deps\wix311-binaries`
```

Comment:
This hsould not be necessary... If so... why so?

Reply:
It is necessary. I don't know why, I have zero knowledge in the rust/cargo stuff but if you don't do this, the build fails as follows:

```
/cygdrive/c/git/IcedTea-Web/launchers
+ mv C:/git/IcedTea-Web/launchers/target/images/icedtea-web C:/git/IcedTea-Web/launchers/target/images/icedtea-web-2.0.0-win.bin
+ isWindows
++ uname
+ [[ CYGWIN_NT-10.0 == *\N\T* ]]
+ return 0
+ which cargo
/cygdrive/c/Users/vippy/.cargo/bin/cargo
+ '[' '!' xC:/msi-deps/wixgen.jar = x -a '!' x/cygdrive/c/msi-deps/wix311-binaries/ = x ']'
+ splitVersion 2.0.0
+ PACKAGE_VERSION=2.0.0
++ echo 2.0.0
++ cut -d. -f 1
+ MAJOR_VERSION=2
++ echo 2.0.0
++ cut -d. -f 2
+ MINOR_VERSION=0
++ echo 2.0.0
++ cut -d. -f 3
++ sed 's/[^0-9]*//g'
+ MICRO_VERSION=0
+ sed -e 's|../win-installer|C:/git/IcedTea-Web/launchers/win-installer|g' -e 's|[@]PACKAGE_VERSION[@]|2.0.0|g' -e 's|[@]MAJOR_VERSION[@]|2|g' -e 's|[@]MINOR_VERSION[@]|0|g' -e 's|[@]MICRO_VERSION[@]|0|g' -e 's/[@]PACKAGE_VERSION[@]/2.0.0/g' C:/git/IcedTea-Web/launchers/win-installer/installer.json.in
++ imageName win.bin
++ echo icedtea-web-2.0.0-win.bin
+ /cygdrive/c/Java/AdoptOpenJDK/jdk-13.0.0.33-hotspot/bin/java -jar C:/msi-deps/wixgen.jar C:/git/IcedTea-Web/launchers/target/images/icedtea-web-2.0.0-win.bin -c C:/git/IcedTea-Web/launchers/target/tmp/itw-installer.json -o C:/git/IcedTea-Web/launchers/target/tmp/itw-installer.wxs
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.sun.xml.bind.v2.runtime.reflect.opt.Injector (file:/C:/msi-deps/wixgen.jar) to method java.lang.ClassLoader.defineClass(java.lang.String,byte[],int,int)
WARNING: Please consider reporting this to the maintainers of com.sun.xml.bind.v2.runtime.reflect.opt.Injector
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
+ pushd C:/git/IcedTea-Web/launchers/target/tmp
/cygdrive/c/git/IcedTea-Web/launchers/target/tmp /cygdrive/c/git/IcedTea-Web/launchers
+ /cygdrive/c/msi-deps/wix311-binaries//candle.exe /nologo itw-installer.wxs
./build.sh: ligne 117: /cygdrive/c/msi-deps/wix311-binaries//candle.exe: No such file or directory

```

Reply:
This error message can be circumvented by setting the `WIX_TOOLSET_DIR` variable, which is otherwise by default set to `/cygdrive/c/msi-deps/wix311-binaries/` [here](https://github.com/AdoptOpenJDK/IcedTea-Web/blob/master/launchers/configure.sh#L169). The default value may be outdated or just plain wrong.

Reply:
@AurumTheEnd @judovana How can we solve this issue? Just remove the line? Add an info about the correct value of the env variable?

Reply:
Just removing the line wouldn't help anyone. As the `WIX_TOOLSET_DIR` is supposed to be the intended way and just isn't clearly marked anywhere right now – your documentation should do that.

Reply:
@don-vip can you take care of that change?